.travis.yml

```diff
@@ -111,3 +124,18 @@ script:
     else
       ctest -V
     fi
+  - |
+    if [[ "${DOCUMENTATION}" == "true" ]]; then
+      cd ..
+      doxygen doxy/doxygen.config
+      # Suppress output to avoid leaking the token when the command fails
+      git clone https://arximboldi:${GH_TOKEN}@github.com/ableton/atria --depth 1 --branch=gh-pages gh-pages &>/dev/null
```

Comment:
Why do we need another clone, and why from your personal clone? We already have a wc. You could do something like `git worktree add ../gh-pages gh-pages`